.travis.yml

```diff
@@ -57,14 +66,18 @@ matrix:
       language: generic
       env:
         - MB_PYTHON_VERSION=3.5
-        - NP_BUILD_DEP=numpy==1.9.3
-        - NP_TEST_DEP=numpy==1.9.3
     - os: osx
       language: generic
       env:
         - MB_PYTHON_VERSION=3.6
         - NP_BUILD_DEP=numpy==1.11.3
         - NP_TEST_DEP=numpy==1.11.3
+    - os: osx
+      langugage: objective-c
```

Comment:
Interesting. Why does it need `objective-c` here and use `generic` in the others?

Reply:
I don't think it does. I had meant to change that back to generic.

Reply:
since `language: generic` is at the top now, I guess we don't need to list anything for language on these OS X cases.